src/components/Typography/Typography.less

```diff
@@ -0,0 +1,58 @@
+@import (reference) '../../styles/variables.less';
+@import (reference) '../../styles/mixins.less';
+
+.@{prefix}-Typography {
+
+	// general text
+	&-p {
+		font-family: @fontFamily;
+	}
+	// text for tables
+	&-tabular {
+		font-family: @font-family-tabular;
+	}
+
+	// headings
+	&-h1,
+	&-h2,
+	&-h3 {
+		margin: 0;
+		font-weight: @font-weight-medium;
+		text-rendering: optimizelegibility;
+	}
+	&-h1 {
+		font-size: 24px;
+		line-height: 28px;
```

Comment:
Do we want all these explicit line-heights or can we rely on master.less to set it to 1.25? These px values seem pretty close to 1.25 x font-size

Reply:
I just pulled in the master.less values for h1